libcudacxx/test/libcudacxx/std/utilities/tuple/tuple.general/ignore.pass.cpp

```diff
@@ -10,17 +10,39 @@
 
 // constexpr unspecified ignore;
 
-// UNSUPPORTED: c++98, c++03
-
 #include <cuda/std/cassert>
 #include <cuda/std/tuple>
 
 #include "test_macros.h"
 
+static_assert(cuda::std::is_trivial<decltype(cuda::std::ignore)>::value, "");
```

Comment:
for the record, `is_trivial` is almost never what you want, and in fact the whole notion of trivial types in the standard is going away. typically what you really want is either `is_trivially_copyable` or `is_trivially_constructible` or both.

Reply:
This is taken from `libc++`. I tried to find out if it is actually required to be trivial by the standard but didn't anything said about it. `libstdc++` does not even implement this check in their tests, but the implementated type is trivial.

The most important thing is that it is trivially constructible. I would change the check to `is_trivially_constructible` if you agree.

Reply:
I would like to have the `is_trivially_default_constructible` and an `is_empty` check